src/test/regress/expected/brin_ao.out

```diff
@@ -466,9 +466,15 @@ CONTEXT:  SQL function "brin_summarize_new_values" statement 1
 SELECT brin_summarize_new_values('tenk1_unique1'); -- error, not a BRIN index
```

Comment:
can we change the commit title to `Revert: "Vacuum auxiliary TOAST should not be dispatched."`

Reply:
which can make cherry-pick gp more suitable..

Reply:
It makes sense